gpm_api/io/checks.py

```diff
@@ -157,15 +157,15 @@ def check_time(time):
     if isinstance(time, np.ndarray):
         if np.issubdtype(time.dtype, np.datetime64):
             if time.size == 1:
-                time = time.astype("datetime64[s]").tolist()
+                time = time.astype("datetime64[us]").tolist()
```

Comment:
Why this change? 
We aim for the output to be a `datetime.datetime` object with second accuracy.

Reply:
@evanjt

Reply:
I have reverted the change